tyr/servers/server.py

```diff
@@ -43,7 +47,7 @@ def __init__(self, group=None, server_type=None, instance_type=None,
                  block_devices=None, chef_path=None, subnet_id=None,
                  dns_zones=None, ingress_groups_to_add=None,
                  ports_to_authorize=None, classic_link=False,
-                 add_route53_dns=True):
+                 add_route53_dns=True, platform=None, use_latest_ami=False):
```

Comment:
You'll need to add these arguments to all the classes which inherit from `Server`, otherwise they won't be exposed. See [this][0] and [this][1].

[0]: https://github.com/hudl/Tyr/issues/117
[1]: https://github.com/hudl/Tyr/pull/115#issuecomment-232234407